<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationWidget</class>
 <widget class="QWidget" name="TransformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="transformation_layout">
    <item row="1" column="1">
     <widget class="QPushButton" name="down_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="translation_button">
      <property name="text">
       <string>Translation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="scaling_button">
      <property name="text">
       <string>Scaling</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="rotation_button">
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="delete_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="up_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="transformation_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
